runtime/Makefile

```diff
@@ -37,8 +37,8 @@ NATIVE_C_SOURCES := $(addsuffix .c, \
   dynlink clambda_checks afl bigarray \
   memprof domain skiplist codefrag)
 
-GENERATED_HEADERS := caml/opnames.h caml/version.h caml/jumptbl.h build_config.h
-CONFIG_HEADERS := caml/m.h caml/s.h
+GENERATED_HEADERS := caml/opnames.h caml/jumptbl.h build_config.h
+CONFIGURED_HEADERS := caml/m.h caml/s.h caml/version.h
```

Comment:
It took me a second or two to realise what was changing here. It's very minor, but might it be worth a comment above: `# $(GENERATED_HEADERS) are built by make; $(CONFIGURED_HEADERS) are built by configure`?